src/main/java/core/basesyntax/SquareSupplier.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+public class SquareSupplier {
+    private static final RandomGeometricValue randomSize = new RandomGeometricValue();

```

Comment:
The field 'randomSize' should not be a constant. When you define a constant it should be named in uppercase.